.vagrant.yml

```diff
@@ -0,0 +1,31 @@
+# QI Configuration file, see: https://github.com/amatas/vagrant-gpii-ci
+---
+
+env:
+  vms:
+    linux:
+      cpu: 2                   # number of cpus
+      memory: 2048             # amount of RAM memory (in Mb)
+      clone: true              # use the linked_clone Vagrant feature
+      autostart: true
+      box: inclusivedesign/fedora27
```

Comment:
Could this be done with the smaller <code>inclusivedesign/centos7</code> image, or are there GUI-related requirements in this workflow that call for the use of Fedora instead?

Reply:
Not sure, I can give it a try in the morning.

Reply:
Or tonight.  Anyway, it worked fine once I updated to use `yum` for various installs.